### PR TITLE
Update manage-creation-of-groups.md

### PR DESCRIPTION
**Pull request by msbemba**
GA can only create groups via EAC, O365 Admin center, Planner and SPO. Teams and Outlook/owa does not let them create it when the restrictions are in place.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4218

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by MikePlumleyMSFT** [comment]
@ArunMSFT can you verify?

File: microsoft-365/solutions/manage-creation-of-groups.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ When you limit who can create a group, it affects all services that rely on grou
 - Power BI (classic)
 - Project for the web / Roadmap
 
-The steps in this article won't prevent members of certain roles from creating Groups. Office 365 Global admins can create Groups via any means, such as the Microsoft 365 admin center, Planner, Teams, Exchange, and SharePoint Online. Other roles can create Groups via limited means, listed below.
+The steps in this article won't prevent members of certain roles from creating Groups. Office 365 Global admins can create Groups via  Microsoft 365 admin center, Planner,  Exchange, and SharePoint Online. Other roles can create Groups via limited means, listed below.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/manage-creation-of-groups.md]:
```suggestion
The steps in this article won't prevent members of certain roles from creating Groups. Office 365 Global admins can create Groups via the Microsoft 365 admin center, Planner, Exchange, and SharePoint Online. Other roles can create Groups via limited means, listed below.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @msbemba - please commit these further changes.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks.

**Comment by MikePlumleyMSFT** [comment]
@ArunMSFT can you verify?

**Comment by yogkumgit** [comment]
@ArunMSFT Kindly verify and merge this PR. Thanks.

**Comment by ArunMSFT** [comment]
Mike - Please merge, thanks.

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Pull request merged**